lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -342,8 +342,8 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 17, 8),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 17, 8),
+        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 18, 8),
+        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 18, 8),
         # split mongo: 3 queries, regardless of thread response size.
         (ModuleStoreEnum.Type.split, 1, 3, 3, 17, 8),
```

Comment:
@cpennington OK, the difference is that with split, creating the course (which is outside of the context manager) ends up triggering the disabled_xblocks check, the result of which is then cached. I believe this is because of building up the course structure.

With old mongo, the first query that triggers disabled_xblocks is within the context manager verifying the number of queries. I have added a comment to the test explaining this.